utils/python/CIME/XML/env_batch.py

```diff
@@ -11,10 +11,12 @@
 
 class EnvBatch(EnvBase):
 
-    def __init__(self, case_root=os.getcwd(), infile="env_batch.xml"):
```

Comment:
Why not just put the case_root initialization in EnvBase and leave it out of the others?